app/public/BreakName.js

```diff
@@ -0,0 +1,51 @@
+var name = 'MIsa';
+
+console.log(checkVowelPosition(name));
+
+// 0: returns an array of single letters
+function stringToArray(string){
+  return string.toLowerCase().split('');
+}
+
+// 1: returns an array containing indeci of vowels
+function checkVowelPosition(name){
+  var vowelPosition = [];
+  var array = stringToArray(name);
+  array.forEach(function(letter) {
+    if (isVowel(letter)) vowelPosition.push(array.indexOf(letter));
+  });
+  return vowelPosition;
+}
+
+// helper functions
```

Comment:
instead of having comments, should we mark the helper functions as private by using an underscore?
eg `function _includeNa` ?